app/src/main/java/com/pajato/android/gamechat/exp/ShowExpListFragment.java

```diff
@@ -32,14 +34,18 @@
         logEvent("onClick (showExp)");
     }
 
-    /** Set the layout file. */
-    @Override public int getLayout() {return R.layout.fragment_game_no_games;}
-
     /** Satisfy the base game fragment contract with a nop message handler. */
     @Override public void messageHandler(final String message) {}
 
+    @Override
```

Comment:
Another doc comment issue plus an @Override all by itself.  When an annotation is short and sweet and putting it on the same line as the code it annotates makes for more concise code with no loss of clarity.